index.mjs

```diff
@@ -0,0 +1,27 @@
+export default workFn => {
+	const hasIdleCallback = typeof requestIdleCallback==='function'
```

Comment:
Yeah, this is more sensible than the window/global boilerplate I pasted in originally.